src/programmatic_policy_learning/envs/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Public API for env utilities (adapters, helpers)"""
+
+from .utils.gym_to_gymnasium import GymToGymnasium
```

Comment:
I think this will go away once we move `GymtoGymnasium` to `prpl-utils`, but in general I recommend using absolute imports everywhere to avoid various headaches